aml-development/ozp-backend
# feat(profile): add avatar to read/write profile view

AMLOS-723

**Description**
As a user, I would like to be able to set my avatar.

**Acceptance Criteria**
User profile page shows the user's avatar
User can change their avatar in the edit profile page
 
**How to test code**    
Pull `profile_avatar` from `ozp-backend`
Pull `profile_avatar` from `frontend`

- Go to the edit profile page as any user.  Click on _Change Avatar_
- Select an image as an avatar and hit _Save_.  Test to see if the avatar updates and the profile view has the new avatar image (both edit profile and normal profile views)
- Test that updating the image removes the old image and replaces it with the new one (both on the frontend and backend)
- Select _Remove_ to remove the current user's avatar, if they have one.  Test that the image is actually removed.

**Please Review**     
@aml-development/ozp-backend-team    

**Checklist**
